doc/users/next_whats_new/updated_mathtext.rst

```diff
@@ -6,9 +6,9 @@ Supports use of bold-italic font style in mathtext using the ``\mathbfit{}`` com
 To change font to bold and italic enclose the text in a font command as
 shown:
 
-.. code-block::
+.. code-block:: python
```

Comment:
Above, this is `none`. I guess both works, but maybe should be consistent?

Reply:
Yes, except that maybe the point of those code blocks is not really Python, but LaTeX, represented as a Python string. So maybe we do not want Python string highlighting? But either will do for sure. Including Python.